docs/schema-packages.txt

```diff
@@ -1,3 +1,6 @@
+SPDX-FileCopyrightText: 2024 Red Hat, Inc
+SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
Please add the hash sign at the beginning of those lines to mark them as comments.